sdk/lib/crt/except/arm/chkstk_asm.s

```diff
@@ -15,6 +15,7 @@
 
     LEAF_ENTRY __chkstk
     __assertfail
+    lsl r4, r4, #2
```

Comment:
Can you add a comment describing inputs and outputs to this function? Using r4 for the result seems odd.

Reply:
The incoming r4 contains the number of dwords to allocate, you convert that to bytes then return...

Reply:
Ah, the code generated to call it looks like this:
```
movw r4, #constant
bl __chkstk
sub.w sp, sp, r4
```

... which is why this part is needed even with the rest of the functionality implemented. Got it.